subtests/docker_cli/run_sigproxy/run_sigproxy.py

```diff
@@ -73,9 +73,9 @@ def _init_container_attached(self, name):
         else:
             subargs = []
         subargs.append(name)
-        container = AsyncDockerCmd(self, 'attach', subargs)
-        self.sub_stuff['container_cmd'] = container  # overwrites finished cmd
-        container.execute()
```

Comment:
R0204: 76,8: sigproxy_base._init_container_attached: Redefinition of container type from dockertest.dockercmd.DockerCmd to dockertest.dockercmd.AsyncDockerCmd